themes/login-forms/templates/Security.ss

```diff
@@ -11,14 +11,24 @@
         <meta name="color-scheme" content="light dark" />
         <% require css("silverstripe/admin: client/dist/styles/bundle.css") %>
         <% require css("silverstripe/login-forms: client/dist/styles/bundle.css") %>
+        <% require javascript('silverstripe/admin: thirdparty/jquery/jquery.js') %>
```

Comment:
Including javascript from the admin thirdparty dir, which is intended for the CMS, not the frontend, is a bad idea.  It's likely that this will break in the future without us realising.  We generally don''t like the dependencies in the thirdparty dir and they don't recieve security updates and we'll refactor them out to be installed via npm/yarn instead.  Recent example of this https://github.com/silverstripe/silverstripe-userforms/pull/1045

You'll need to change this to get jquery / tooltip, etc, to install via yarn and then included in the frontend js bundle